SECURITY.md

```diff
@@ -0,0 +1,11 @@
+## Report a security issue
+
+The tokio project team welcomes security reports and is committed to providing prompt attention to security issues. Security issues should be reported privately via [security@tokio.rs](mailto:security@tokio.rs).
+
+## Vulnerability coordination
+
+Remediation of security vulnerabilities is prioritized by the project team. The project team coordinates remediation with third-party project stakeholders via [Github Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories).
```

Comment:
Should there be a definition of what it means to be a "third-party project stakeholder"? Should there be a description of how a third-party project that depends on Tokio would become involved in this process?

Reply:
This can include (non-exclusively): the issue-reporter/bug-finder, [a subset of] affected direct or indirect users of tokio, and maintainers of upstream dependencies (in case the issue is actually in a lib tokio uses, for example). I will include these as examples.

PS In case you are interested, these types of entities are adapted from the [FIRST coordination guideline](https://www.first.org/global/sigs/vulnerability-coordination/multiparty/guidelines-v1.0).